python/ray/_raylet.pyx

```diff
@@ -2907,18 +2907,23 @@ cdef class OldGcsClient:
         return result
 
     @_auto_reconnect
-    def get_all_job_info(self, timeout=None) -> Dict[JobID, JobTableData]:
+    def get_all_job_info(self, timeout=None, **kwargs) -> Dict[JobID, JobTableData]:
```

Comment:
wonder if we can make the fields just defined here, instead of **kwargs:

def get_all_job_info(self, query_job_info_field, c_query_is_running_tasks_field, timeout=None) -> Dict[JobID, JobTableData]:

Reply:
+1 

Let's err on the side of properly codifying params

Reply:
Good suggestion, have updated to use explicit field definitions instead.